app/adapters/crate.js

```diff
@@ -25,6 +25,18 @@ export default class CrateAdapter extends ApplicationAdapter {
     return `${baseUrl}/${crateName}`;
   }
 
+  /** Adds a `reason` query parameter to the URL, if set in the `adapterOptions`. */
+  urlForDeleteRecord(id, modelName, snapshot) {
+    let url = super.urlForDeleteRecord(...arguments);
+
+    let reason = snapshot.adapterOptions.reason;
+    if (reason) {
+      url += `?reason=${encodeURIComponent(reason)}`;
```

Comment:
@Turbo87 🙋  If I understand correctly, the query string is named `message` at the backend https://github.com/rust-lang/crates.io/blob/19c80e04bd4446d96cc9ebbe7dfe4ac87c4345ce/src/controllers/krate/delete.rs#L24-L29, but here we are sending it as `reason`. I think this might be a mistake?

Reply:
damn, you're right... 🤦‍♂️